installation/scripts/minikube.sh

```diff
@@ -101,6 +102,18 @@ function checkMinikubeVersion() {
     fi
 }
 
+function checkKubectlVersion() {
+    local clientVersion=$(kubectl version --short | grep "Client" | awk '{print $3}')
+    local clientVersionMinor=$(echo $clientVersion | awk -F'[(.)]' '{print $2}')
+    local kubectlCliVersionMinor=$(echo $KUBECTL_CLI_VERSION | awk -F'[(.)]' '{print $2}')
+    local versionDifference=$(( clientVersionMinor - kubectlCliVersionMinor ))
+
+    if [[ $versionDifference -ne 1 ]] && [[ $versionDifference -ne 0 ]] && [[ $versionDifference -ne -1 ]]; then
```

Comment:
Are you sure you need as many as three conditions here? Perhaps something like AbsoluteValueOfVersionDiff <= 1 would do the trick, huh?